src/mds/MDCache.cc

```diff
@@ -13488,6 +13488,13 @@ bool MDCache::dump_inode(Formatter *f, uint64_t number) {
   return true;
 }
 
+bool MDCache::dump_dir(Formatter *f, CDir *dir, bool dentry_dump) {
+  f->open_object_section("dir");
+  dir->dump(f, dentry_dump ? CDir::DUMP_ALL : CDir::DUMP_DEFAULT);
+  f->close_section();
+  return true;
+}
```

Comment:
code looks good, just a question: is there any reason for this function to have `bool` return type? I think `void` would do too

Reply:
> code looks good, just a question: is there any reason for this function to have `bool` return type? I think `void` would do too

I have modified it. At first I referred to `dump_inode` and the type of its return value is boolean.